common/src/main/java/com/wynntils/core/features/overlays/OverlayPosition.java

```diff
@@ -4,24 +4,80 @@
  */
 package com.wynntils.core.features.overlays;
 
-// TODO: This class is very basic atm
 public class OverlayPosition {
-    // Top-Left point of Overlay
-    private int x;
-    private int y;
 
-    private OverlayPosition() {}
+    private final int verticalOffset;
+    private final int horizontalOffset;
```

Comment:
i'm not sure if all the fields here should be final - having to create a new position object and reassign it in the right places seems unnecessary. plus, if we intend on a gui placement screen that allows users to drag overlays around, being able to update these values in real-time and use them for rendering in that screen as well